Packs/CyberArkPAS/Integrations/CyberArkPAS/CyberArkPAS.yml

```diff
@@ -0,0 +1,913 @@
+      required: true
+      secret: false
+    - default: false
+      description: MM\DD\YY or empty if there is no expiration date.
+      isArray: false
+      name: membership_expiration_date
+      required: false
+      secret: false
+    - default: false
+      description: |-
+        User’s permissions in the Safe.
+        Optional values:
+        UseAccounts, RetrieveAccounts, ListAccounts, AddAccounts, UpdateAccountContent, UpdateAccountProperties, InitiateCPMAccountManagementOperations, InitiateCPMAccountManagementOperations, SpecifyNextAccountContent, RenameAccounts, DeleteAccounts, UnlockAccounts, ManageSafe, ManageSafeMembers, BackupSafe, ViewAuditLog, ViewAuditLog, ViewSafeMembers, RequestsAuthorizationLevel, AccessWithoutConfirmation, CreateFolders, DeleteFolders, MoveAccountsAndFolders
+      isArray: true
+      name: permissions
+      required: false
+      secret: false
+    - default: false
+      description: The name of the Safe to add a member to.
+      isArray: false
+      name: safe_name
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: '0'
+      description: "Requests Authorization Level.\n0 – cannot authorize\t\n1 – authorization\
+        \ level 1\t\n2 – authorization level 2"
+      isArray: false
+      name: requests_authorization_level
+      predefined:
+      - '0'
+      - '1'
+      - '2'
+      required: false
+      secret: false
+    deprecated: false
+    description: Update an existing Safe member.
+    execution: false
+    name: cyberark-pas-update-safe-member
+  - arguments:
+    - default: false
+      description: The name of the Safe to add a member to.
+      isArray: false
+      name: safe_name
+      required: true
+      secret: false
+    - default: false
+      description: The name of the user to add as a Safe member.
+      isArray: false
+      name: member_name
+      required: true
+      secret: false
+    deprecated: false
+    description: Remove a specific member from a Safe.
+    execution: false
+    name: cyberark-pas-delete-safe-member
+  - arguments:
+    - default: false
+      description: The name of the account.
+      isArray: false
+      name: account_name
+      required: true
+      secret: false
+    - default: false
+      description: The name or address of the machine where the account will be used.
+      isArray: false
+      name: address
+      required: true
+      secret: false
+    - default: false
+      description: The platform assigned to this account.
+      isArray: false
+      name: platform_id
+      required: true
+      secret: false
+    - default: false
+      description: The Safe where the account will be created.
+      isArray: false
+      name: safe_name
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: password
+      description: The type of password.
+      isArray: false
+      name: secret_type
```

Comment:
should we add a default value here?
there are a lot of args to each and every command

Reply:
it's not mandatory so I don't think we should.